Velaptor/UI/MouseMoveEventArgs.cs

```diff
@@ -0,0 +1,39 @@
+// <copyright file="MouseMoveEventArgs.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace Velaptor.UI;
+
+using System.Diagnostics.CodeAnalysis;
+using System.Drawing;
+
+/// <summary>
+/// Holds information about the mouse position over a control.
+/// </summary>
+public readonly record struct MouseMoveEventArgs
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="MouseMoveEventArgs"/> struct.
+    /// </summary>
+    /// <param name="globalPos">The global position of the mouse.</param>
+    /// <param name="localPos">The local position of the mouse.</param>
+    [SuppressMessage(
+        "StyleCop.CSharp.DocumentationRules",
+        "SA1642:Constructor summary documentation should begin with standard text",
+        Justification = "Reported incorrectly.  This is a struct, not a class")]
+    public MouseMoveEventArgs(Point globalPos, Point localPos)
+    {
+        GlobalPos = globalPos;
+        LocalPos = localPos;
+    }
+
+    /// <summary>
+    /// Gets the position of the mouse relative to the top left corner of the window.
+    /// </summary>
+    public Point GlobalPos { get; }
+
+    /// <summary>
+    /// Gets the position of the mouse relative to the top left corner of the control.
+    /// </summary>
+    public Point LocalPos { get; }
```

Comment:
It is fine the way it is.  It is what it says that it is.

The global position represents the position of the mouse in the window.  Controls live in this window space. So when you talk about a control position, it is the position of the control in its own "world" which is the window.  

Local position is the position of the mouse _**RELATIVE**_ to the top left corner of the control itself, not the window.

<img width="420" alt="image" src="https://github.com/KinsonDigital/Velaptor/assets/105235056/e403500a-b5fb-489e-9610-9d8ca39e5e99">

In this image below, the distance in orange is the horizontal distance relative to the left side of the window.  If the mouse was touching the left side of the button, this would be the _**global**_ mouse position.  

The yellow is the distance from the left side of the button to the mouse cursor.  This would be the _**local**_ mouse position.